vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
-these APIs to create pages, create page templates or to add widgets to
-pages. The APIs can be used to support automation, testing, or
-integrations with external systems.
+5. (Optional) Reset the deployment `spec.replicas` back to 1 if it was scaled to "0" in the previous steps. Save the deployment to update.
 
-### API structure
+### Disable Swagger UI
 
-All of the APIs share a common top level structure. Each response will
-contain a top level entry for `errors`, `metadata`, and `payload`.
+Repeat the steps above, but in step 4, remove `swagger` from the value list.
 
-The `errors` will always contain code and a message string indicating an
-error condition in the request. The `metadata` section is used for
-paging, sorting, filtering and data that is distinct from the body. The
-body of each response is included in the `payload` section of the
-response and varies according to each API.
+## Find Your Client Secret
+You'll need your client credentials to execute the Entando APIs. 
 
-### Models
+1. Login into your Keycloak Administration Console. To find the Kubernetes credentials, see the [Entando Identity Management System](./identity-management.md) page.
 
-All of the model classes returned by the Entando core are annotated so that the model definition is included in the Swagger documentation. At the bottom of the Swagger page all of the model classes returned by the API endpoints can be found.
+2. From the left nav, go to `Clients`
 
-## Tutorial
+3. Select the desired client (e.g. in a quickstart environment this is `quickstart`)
 
-1. Access your application Swagger UI as discussed above
+4. Click on the `Credentials` tab to retrieve the secret 
 
-2. Click on the `Authorize` button in the upper right corner
+## Access the APIs on Swagger
+1.  To see the APIs, go to: 
+``` sh
+http://[YOUR-HOST-NAME]/entando-de-app/api/swagger-ui.html
+```
 
-3. Enter the client id and client secret in the open window and click `Authorize`
+2. Click on the Authorize button in the upper right corner.
 
-4. If you are redirected to the Entando login page, log in with your credentials (default are `admin`/`adminadmin`)
+3. Enter the client id and secret and click Authorize.
 
-5. You will be redirected to the Swagger UI page, now authenticated
+4. You will be prompted to log in to your Keycloak instance as an admin user if you have not already done so. The defaults are admin/adminadmin. 
 
-6. Use the **Try it out** button on the APIs
+5. You will be redirected to the authenticated Swagger UI page. Select an API to see the methods in the drop-down list and click the `Try it out` button.  
+Example:
+   1. Scroll down to `widget-controller`
 
-    -   Scroll to `widget-controller`
+   2. Select the `GET` method
 
-    -   Select the blue GET row
+   3. Click `Try it out`
 
-    -   Select **Try it out**
+   4. See the results in the window below. You should see a Server response with Code 200 and full response body.
```

Comment:
"The results are visible in the window below" or "view the results..." (a bit less aggressive and avoids a repeat of the word "see")

"see a Server response with Code 200 and full response body"... why is server capitalized? if code is capitalized "Code 200" should be in quotes to indicate that's the actual value you see (conversely code 200 implies you'll see just the number 200)

"You should see a Server response with Code 200 and full response body"... i feel like this could be rephrased for clarity and to avoid two "responses"

Reply:
not sure why server was capped??
Code 200 is like Error 404, does it need backticks?? In this case, don't think it's necessary
edited

I actually prefer 'see the results..' it's direct and clear. view sounds too much like sightseeing.

Reply:
i don't think backticks, only quotes because it's a name

we should def go for the sightseeing angle. maybe we could start selling merch ;)

Reply:
It's not a name, it's a resultant code and in HTTP protocols, it's a common response that really doesn't need special citing

Reply:
it's the name of a code. i'm fine with capitalized and no quotes, but we have quoted or otherwise formatted codes elsewhere in the documentation so this is more about consistency. should we say going forward to not format stuff like this? i actually think we should because it gets really busy with backticks/quotes and like you said, it's not necessary